cmd/adapter-rest/startcmd/start.go

```diff
@@ -622,6 +623,20 @@ func constructCORSHandler(handler http.Handler) http.Handler {
 	).Handler(handler)
 }
 
+func initRPAdapterEdgeStores(dbURL string) (persistent, transient storage.Provider, err error) {
+	persistent, err = initEdgeStore(dbURL, rpAdapterPersistentStorePrefix)
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to init edge persistent storage: %w", err)
+	}
+
+	transient, err = initEdgeStore(dbURL, rpAdapterTransientStorePrefix)
```

Comment:
Shouldn't we have option to pass different URLs/different types for persistent and transient storages? Here it looks like we are using same db instances with different store prefix.

Reply:
@rolsonquadras yes, that would be a followup

Reply:
#226